### ISSUE TITLE
Java exceptions appear to be uncaught by crash reporter

### ISSUE THREAD

**Issue by cvan** [MozillaReality/FirefoxReality#1127]
## Hardware
Oculus Go
master (d688580)

## Steps to Reproduce

1. From Android Studio, install a 64-bit Oculus build + run FxR on the Oculus Go.
2. Put the headset on, and refresh the browser page, navigate to an HTTPS site, etc.
3. From Android Studio, change the Build Variant to 32-bit Oculus + run FxR on the Oculus Go.
4. Put the headset on, and refresh the browser page, navigate to an HTTPS site, etc.
5. Notice a crash occurs when navigating to an HTTPS site.
6. Notice FxR reloads without a crash prompt, and the logs don't seem to catch it.

## Context
The HTTPS issue is a GeckoView bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1546361

But it's suspicious that we see this without a crash:

```
04-24 17:01:45.540 12406 12406 I art     : Rejecting re-init on previously-failed class java.lang.Class<androidx.core.view.ViewCompat$OnUnhandledKeyEventListenerWrapper>: java.lang.NoClassDefFoundError: Failed resolution of: Landroid/view/View$OnUnhandledKeyEventListener;` with out a crash.
```

## Error Logs and Stack Traces
See my [**`logcat`**](https://github.com/MozillaReality/FirefoxReality/files/3114626/fxr-full-logs-crash-32.txt).

CC @bluemarvin @MortimerGoro

**Comment by MortimerGoro** [comment]
I also had this problem yesterday while developing, Java stacktraces didn't appear in the logcat

**Comment by MortimerGoro** [comment]
I'm using this snippet in order to get the Java exceptions locally. Otherwise they are hidden by this GV bug making local dev hard.
```
mUiThread.setUncaughtExceptionHandler((t, e) -> {
    Log.e(LOGTAG, "Uncaught exception: " + e.toString());
    e.printStackTrace();
});
```

**Comment by cvan** [comment]
thanks. we should be able to make a PR for this until the bug is fixed in GV. is there a filed GV bug? I couldn't find one on Bugzilla:

- https://bugzilla.mozilla.org/buglist.cgi?quicksearch=geckoview%20exception&list_id=14701697
- https://bugzilla.mozilla.org/buglist.cgi?quicksearch=geckoview%20error&list_id=14701699

**Comment by bluemarvin** [comment]
It's a bug in GV. I have a fix which should be able to land quickly.

**Comment by bluemarvin** [comment]
https://bugzilla.mozilla.org/show_bug.cgi?id=1550185

**Comment by bluemarvin** [comment]
We will need to bump the GV version once it lands in nightly.

**Comment by bluemarvin** [comment]
fixed in #1187
